### PR TITLE
Move scaladoc documentation from dotty.epfl.ch to a guide

### PR DESCRIPTION
**Pull request by julienrf**
Content was taken from commit c8894a0168061ddb1369a426e9bde8d9972c893b

Screenshot from 2021-07-21 15-52-23

### CODE REVIEW

File: scala3/guides.md [hunk 1]
```diff
@@ -18,6 +18,11 @@ guides:
     icon: birthday-cake
     url: "/scala3/guides/tasty-overview.html"
     description: "An overview over the TASTy format aimed at end-users of the Scala language."
+  - title: Scaladoc
+    by: Krzysztof Romanowski, Aleksander Boruch-Gruszeski, Andrzej Ratajczak
```
Comment (julienrf) [inline scala3/guides.md:22]: Please let me know if I missed someone

Comment (BarkingBad) [reply scala3/guides.md:22]:
```suggestion
    by: Krzysztof Romanowski, Aleksander Boruch-Gruszecki, Andrzej Ratajczak
```

@KacperFKorban @pikinier20, have you contributed to these docs?

Comment (KacperFKorban) [reply scala3/guides.md:22]: I think that I haven't...yet

**Comment by prolativ** [comment]
I think we should ask @BarkingBad for comments (but it seems I can't assign him as a reviewer explicitly)

File: scala3/guides.md (hunk 1 above)
Comment (prolativ) [reply scala3/guides.md:22]: `Aleksander Boruch-Gruszecki`

File: _overviews/scala3-scaladoc/index.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+layout: multipage-overview
+title: Scaladoc
+partof: scala3-scaladoc
+num: 1
+next-page: docstrings
+---
+
+![scaladoc logo]({{ site.baseurl }}/resources/images/scala3/scaladoc/logo.png)
```
Comment (prolativ) [inline _overviews/scala3-scaladoc/index.md:9]: This seems to be the old logo. Here's the new one https://github.com/lampepfl/dotty/blob/master/scaladoc/resources/dotty_res/images/scaladoc_logo.svg

Comment (julienrf) [reply _overviews/scala3-scaladoc/index.md:9]: Changed

**Comment by prolativ** [comment]
LGTM but the CI is failing. Is this somehow connected with changing the logo image?

**Review by BarkingBad — COMMENTED** [review]
For some reason all headers 4 and 5 are in upper case

obraz

Can we do something with that?

**Comment by BarkingBad** [comment]
Also, what is the correct navigation from the root page to the docs? I had to hardcode the link as I couldn't reach the docs

**Comment by julienrf** [comment]
@BarkingBad good point about the headers of level 4 and 5. I’ve added a commit that changes the style, it now looks like the following:

Screenshot from 2021-07-23 10-46-51

Hopefully this is not breaking other things…

> Also, what is the correct navigation from the root page to the docs? I had to hardcode the link as I couldn't reach the docs

index -> scala 3 guides -> scaladoc

**Comment by BarkingBad** [comment]
> index -> scala 3 guides -> scaladoc

Oh yes, I must have missed something. Thanks for fixing styles

**Review by BarkingBad — APPROVED** [review]

**Comment by julienrf** [comment]
Thank you guys for the review!

**Pull request merged**